dynawo/sources/Modeler/Common/DYNSubModel.cpp

```diff
@@ -965,6 +963,157 @@ SubModel::addParameterCurve(shared_ptr<curves::Curve>& curve) {
   curve->setNegated(false);
 }
 
+void
+SubModel::printLocalInitParametersValues() const {
+  const boost::unordered_map<std::string, ParameterModeler>& params = getParametersDynamic();
+  std::set<std::string> sortedParams;
+  for (boost::unordered_map<std::string, ParameterModeler>::const_iterator it = params.begin(), itEnd = params.end();
+      it != itEnd; ++it) {
+    sortedParams.insert(it->first);
+  }
+  if (!params.empty()) {
+    Trace::debug(Trace::parameters()) << "------------------------------" << Trace::endline;
+    Trace::debug(Trace::parameters()) << "SubModel " << name()  << " parameters after initialization"<< Trace::endline;
```

Comment:
Can you add "local" initialization?

Reply:
Or is it global init? if so method name is misleading..

Reply:
Solved